.pylintrc

```diff
@@ -12,6 +12,8 @@ disable=
     bad-continuation,           # clashes with black
     too-few-public-methods,     # triggers when inheriting
     ungrouped-imports,          # clashes with isort
+    no-self-use,                # clashes with E0213
+    duplicate-code,             # duplicate test code - can't be ignored inline (https://github.com/PyCQA/pylint/issues/214)
```

Comment:
I'd rather disable both of these on a function/line level than disable them from the whole project. I think I understand the second issue with the second one but where are these occurring in the PR?

Reply:
where's the 'no-self-use' occurring? cant seem to find it looking at the PR

Reply:
yep, removed. parametrized fixture instead in the test module so this change is not necessary